dotcom-rendering/src/client/bootCmp.ts

```diff
@@ -81,6 +81,47 @@ const submitConsentEventsToOphan = (renderingTarget: RenderingTarget) =>
 		return submitComponentEvent(event, renderingTarget);
 	});
 
+const submitConsentToOphan = (renderingTarget: RenderingTarget) =>
+	onConsent().then((consentState: ConsentState) => {
+		// eslint-disable-next-line @typescript-eslint/strict-boolean-expressions, @typescript-eslint/no-unnecessary-condition -- Review if this check is needed
+		if (!consentState) return;
```

Comment:
TypeScript knows that consent state will never be falsy, because it’s an object, which is always truthy.